pom.xml

```diff
@@ -33,7 +33,7 @@
 	</licenses>
 	<properties>
 		<!-- Notice: if using java 9 or 10 jres, it is possible to simply use here "8", "9" or "10". But if using java 8 jre, "8" is not an allowed value but only "1.8"-->
-		<jdk.version>1.8</jdk.version>
+		<jdk.version>11</jdk.version>
 		<maven.enforcer.jdk-version>1.8</maven.enforcer.jdk-version>
```

Comment:
Might need/want to bump `<maven.enforcer.jdk-version>` too

Reply:
Good catch. Is there any point in keeping them as separate variables? Can they possibly diverge?

Reply:
I can't think of any reason to keep them separate

Reply:
Ok I've merged the variables